<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Join_view</class>
 <widget class="QDialog" name="Join_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color : white;

font: 75 10pt &quot;맑은 고딕&quot;;</string>
     </property>
     <property name="title">
      <string>회원 가입</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="EditName">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
font-weight : bold;</string>
        </property>
        <property name="text">
         <string>이 름</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="EditNum">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
font-weight : bold;</string>
        </property>
        <property name="text">
         <string>학 번</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border : 4px solid;
border-color : blue;
border-radius : 20px;
padding : 5px;

background-color : blue;
color: rgb(255, 255, 255);

font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>중복확인</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
font-weight : bold;</string>
        </property>
        <property name="text">
         <string>캠퍼스</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
font-weight : bold;</string>
        </property>
        <property name="text">
         <string>학 과</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
font-weight : bold;</string>
        </property>
        <property name="text">
         <string>개인 번호</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="EditCampus">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>1 (고성캠퍼스)</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_complete">
        <property name="styleSheet">
         <string notr="true">border : 4px solid;
border-color : blue;
border-radius : 20px;
padding : 5px;

background-color : blue;
color: rgb(255, 255, 255);

font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>가입 완료</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>151</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_cancel">
        <property name="styleSheet">
         <string notr="true">border : 4px solid;
border-color : blue;
border-radius : 20px;
padding : 5px;

background-color : blue;
color: rgb(255, 255, 255);

font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>취소하기</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="EditPrivateNum">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="EditMajor_2">
        <item>
         <property name="text">
          <string>학과선택</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관광경영학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>호텔경영학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>항공경영학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>대명리조트학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>해양심층수학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>외식사업학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>호텔조리학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>경찰학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>행정학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>경영학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>정보보안학과</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>컴퓨터공학과</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
